app/_src/gateway-operator/get-started/kic/create-gatewayclass.md

```diff
@@ -0,0 +1,91 @@
+---
+title: Create a GatewayClass
+content-type: tutorial
+book: kgo-kic-get-started
+chapter: 2
+alpha: true
+---
+
+To use the Gateway API resources to configure your routes, you need to create a `GatewayClass` instance and create a `Gateway` resource that listens on the ports that you need.
+
+{:.note}
+> **Note:** `Gateway` and `ControlPlane` controllers are still `alpha` so be sure
+> to use the [installation steps from this guide](/gateway-operator/{{ page.release }}/get-started/kic/install/)
+> in order to get your `Gateway` up and running.
+
+```yaml
+echo '
+kind: GatewayConfiguration
+apiVersion: gateway-operator.konghq.com/v1alpha1
+metadata:
+  name: kong
+  namespace: default
+spec:
+  dataPlaneOptions:
+    deployment:
+      podTemplateSpec:
+        spec:
+          containers:
+          - name: proxy
+            image: kong:{{ site.data.kong_latest_gateway.ce-version }}
+            readinessProbe:
+              initialDelaySeconds: 1
+              periodSeconds: 1
+  controlPlaneOptions:
+    deployment:
+      podTemplateSpec:
+        spec:
+          containers:
+          - name: controller
+            image: kong/kubernetes-ingress-controller:{{ site.data.kong_latest_KIC.version }}
```

Comment:
@mheap this will work now/today but 1.0.0 does not work fully with KIC 2.12 yet.

Issue: https://github.com/Kong/gateway-operator/issues/1122
PR where CI failed: https://github.com/Kong/gateway-operator/pull/1118

Reply:
What's the ETA on working with 2.12? I'm planning on bumping KIC docs on Mon next week

Reply:
I'm looking into this in https://github.com/Kong/gateway-operator/pull/1118

Reply:
The above is ready and we should be good to go with using 2.12 (in 1.0.1 or so).